### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [ ] bump `invenio-base` dependency

**Comment by SamiHiltunen** [comment]
@jirikuncar bumped up invenio-base to 0.3.0 and rebased

File: invenio_workflows/api.py [hunk 1]
```diff
@@ -54,9 +55,9 @@ def worker(self):
             return import_string('invenio_workflows.workers.%s:%s' % (
                 cfg['CFG_BIBWORKFLOW_WORKER'], cfg['CFG_BIBWORKFLOW_WORKER']))
         except Exception:
-            from flask import current_app
```
Comment (jirikuncar) [inline invenio_workflows/api.py]: it looks like wrong indentation

File: invenio_workflows/utils.py [hunk 2]
```diff
@@ -25,7 +25,13 @@
 
 from flask import current_app, jsonify, render_template
 
+<<<<<<< HEAD
```
Comment (jirikuncar) [inline invenio_workflows/utils.py]: merge leftovers

**Pull request merged**